wkz/watchdogs.py

```diff
@@ -31,7 +31,10 @@ def trigger_device_watchdog():
 def _watch_for_device(path_to_garmin_device: str, path_to_trace_dir: str, delete_files_after_import: bool):
     device_mounted = False
     if Path(path_to_garmin_device).is_dir():
-        sub_dirs = os.listdir(path_to_garmin_device)
+        sub_dirs = []
+        for filename in os.listdir(path_to_garmin_device):
+            if Path(os.path.join(path_to_garmin_device, filename)).is_dir():
+                sub_dirs.append(sub_dirs)
         if len(sub_dirs) == 1 and not device_mounted:
```

Comment:
As far as I remember there was no specific reason why I chose `== 1` (besides that this was the case for my device). Thinking about this now seems too strict, because `FitCollector` will search for the dedicated folder names anyway. How about:
```suggestion
        if len(sub_dirs) > 0 and not device_mounted:
```
This could allow devices to be recognized even if there are multiple directories. With this approach we could even avoid your above `for` loop.

Reply:
I think we still need to check if there is at least 1 directory, which contains the `activity` dir. But I'm not 100% sure. But of course it is better to work when there are multiple directories.